localedb_man.py

```diff
@@ -1093,11 +1094,127 @@ def test(self):
             c.execute(f'SELECT COUNT(*) AS n FROM {self.dbi.pg_schema_weather}.weather;')
             print(c.fetchone().n)            
 
+# ----------------------------------------------------------------------------------------------------------------------
+class MobilitySchema(Schema):
+    """County mobility data is available here: https://data.bts.gov/api/views/w96p-f2qv/rows.csv?accessType=DOWNLOAD
+
+    Data mirror: https://world-modelers.s3.amazonaws.com/data/Trips_by_Distance.csv 
+    (will become stale)
+    """
+
+
+    def add_zero(self, x):
```

Comment:
Not sure if that would be slower or faster, but to make it more succinct we could use [pandas.Series.str.pad](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.str.pad.html)

Reply:
That seems reasonable to me and is cleaner, will give it a go.

Reply:
Cool.

Reply:
Since this is used in a converter I think it's simplest to leave as is:

```
df_full = pd.read_csv(trips_fn, sep=",", 
                              converters={'County FIPS': lambda x: self.add_zero(str(x))},engine='python')
```